.gitignore

```diff
@@ -1,3 +1,117 @@
+
+# Created by https://www.toptal.com/developers/gitignore/api/python,pycharm
```

Comment:
Does it also make sense to add the gitignore for qml and qt? Like [these](https://www.toptal.com/developers/gitignore/api/qml,qt)

Reply:
No, at the moment not, because there are no files related to qml or qt at the moment. Qt is included and abstracted by PyQt. If this will change, I will add it later.